src/shimoku/actions_execution/front_connection.py

```diff
@@ -0,0 +1,38 @@
+from shimoku.utils import IN_BROWSER_PYODIDE
```

Comment:
Avoid library-centric logic (naming in this case)

Reply:
In this case this variable is set by checking if the library exists in the modules, if it exists this flag is set to true, is the only way to test if pyodide exists and is beign executed in the browser, which name do you think would fit better? Maybe 'IN_BROWSER' is clear enough

Reply:
Yes, I think IN_BROWSER could be better